### PR TITLE
Salary work relations

### LINKED ISSUE #203: Pick up WR id from salary
Pick up WR id from salary and find values on server side.

Keep the displayed values there but don't post them.

### LINKED ISSUE #208: When you create a salary select the first possible work relation automaticly

### PR DESCRIPTION
**Pull request by Baxxx**
Closes #203
Closes #208

### CODE REVIEW

**Comment by MrdjanStojimirovic** [comment]
First round review: @MrdjanStojimirovic
Final review with: @stoivo

File: modules/salary/view/record.inc [hunk 1]
```diff
@@ -228,21 +228,31 @@ elseif($_lib['input']->getProperty('action_salary_new'))
         $result_head_employee = $_lib['db']->db_query($query_head_employee);
         $head_employee        = $_lib['db']->db_fetch_object($result_head_employee);
         $accountplan = $accounting->get_accountplan_object($head_employee->AccountPlanID);
+        $work_relation_query = "SELECT * FROM workrelation WHERE AccountPlanID = '".$head_employee->AccountPlanID."' ORDER BY WorkRelationID ASC LIMIT 1";
```
Comment (mladjo2505) [inline modules/salary/view/record.inc]: Choose the latest WR.

**Comment by mladjo2505** [comment]
Change the 'Lagre altinn felter' action to only update the work relation fields on salary.
Add a new indicator when the work relation/altinn fields were last updated and by whom.
On save action update both saved by and saved at for salary and altinn fields but on Save altinn fields action only update the ones for altinn fields.

**Comment by sls-lnluksa** [comment]
First round review: Nikola Lukovic (sls-lnluksa)
Final review with: @stoivo

**Comment by mladjo2505** [comment]
Problem is in:

``` php
$work_relation_query = "SELECT * FROM workrelation WHERE WorkRelationID = '".$_POST['salary_WorkRelationID_'.$SalaryID]."'";
```

since `$_POST['salary_WorkRelationID_'.$SalaryID]` is not sent on `'action_salary_new'`.
The next update on salary table sets all WR related fields to NULL.

**Comment by sls-lnluksa** [comment]
First round review: Nikola Lukovic (sls-lnluksa)
Final review with: @stoivo

**Pull request merged**